src/styles/styles.scss

```diff
@@ -0,0 +1,7 @@
+@import "./blocks/index";
+@import "./blocks/variables";
```

Comment:
Import the 'variables' file before other files to make the variables available for use.